forge/files/src/index.js

```diff
@@ -32,7 +31,7 @@ app.on('window-all-closed', () => {
 });
 
 app.on('ready', async () => {
-  if (isDev) {
+  if (app.isPackaged) {
```

Comment:
@bendemboski should this be `!app.isPackaged`? Seems like dev should be not packaged right? Maybe I am misunderstanding.

Reply:
https://github.com/adopted-ember-addons/ember-electron/pull/1698